lib/bcdice/game_system/SajinsenkiAGuS2E.rb

```diff
@@ -0,0 +1,229 @@
+            [60, 'ミドルシールドを獲得。 →p35'],
+            [61, 'ヘビーシールドを獲得。 →p35'],
+            [62, 'レーダーユニットを獲得。 →p35'],
+            [63, 'ECMユニットを獲得。 →p35'],
+            [64, 'サブブースターを獲得。 →p36'],
+            [65, 'ディフェンスサポートを獲得。 →p36'],
+            [66, 'コンバットサポートを獲得。 →p36'],
+            [67, '大失敗…。大きな損失を出してしまった…。 -5,000cdt'],
+            [68, 'ショットサポートを獲得。 →p36'],
+            [69, 'パワーローダーを獲得。 →p36'],
+            [70, 'サブバッテリーを獲得。 →p36'],
+            [71, 'サブバッテリー+を獲得。 →p36'],
+            [72, 'ファランクスを獲得。 →p36'],
+            [73, 'リアクティブアーマーを獲得。 →p36'],
+            [74, '強化装甲版を獲得。 →p36'],
+            [75, 'ヘビーマシンガンSを獲得。 →p35'],
+            [76, 'ヘビーマシンガンLを獲得。 →p35'],
+            [77, '掘り出し物を発見！25,000cdt以下の【上級外装兵装】一つを獲得！ 上級兵装 (→p37）'],
+            [78..79, '失敗…。かなりの損失を出してしまった…。 -3,000cdt'],
+            [80, '医療用品を獲得！ 調息値+［1］（全PC）'],
+            [81, '大型ソーラーパネルを獲得！ 整備値+［1］（全PC）'],
+            [82, '艦内用の環境設備を獲得！ サポート使用回数+［1］'],
+            [83, 'リニアガンを獲得。 →p37'],
+            [84, 'リニアマシンガンを獲得。 →p37'],
+            [85, 'ジャマ―ユニットを獲得。 →p37'],
+            [86, 'センサー+を獲得。 →p37'],
+            [87, 'パワーローダー++を獲得。 →p37'],
+            [88, 'サブバッテリー++を獲得。 →p37'],
+            [89, 'フレームカバーを獲得。 →p37'],
+            [90, '空振り……何の成果も得られなかった…。 0cdt'],
+            [91..92, '良質なA:GuSのパーツを獲得！機体の改造や予備パーツとして使えそうだ！ 改造Lv+［1］（上限：3Lv）'],
+            [93..95, 'A:GuSのパーツを獲得！機体の改造や予備パーツとして使えそうだ。 改造Lv+［1］（上限：1Lv）'],
+            [96..98, '多少傷ついているが、A:GuS1機を獲得！→10,000cdtで売却可能→10,000cdt支払えば補修して取得が可能。 （→p30～33）（→w23）'],
+            [99, 'A:GuS1機をほぼ完全な状態で獲得！ （→p30～33）（→w23）'],
+            [100, '掘り出し物を発見！25,000cdt以下の【上級携行兵装】一つを獲得！ 上級兵装 (→p37）'],
+          ]
+        ),
+        "DEF" => DiceTable::RangeTable.new(
+          "赤字ペナルティー表",
+          "1D9",
+          [
+            [0..1, '解雇 クルー1名を失う。10,000cdtを得る。'],
+            [2..3, '劣化 任意のチーム能力一つは-1Lv。10,000cdtを得る。'],
+            [4..5, '借金 次回の維持費が+20,000cdt。10,000cdtを得る。'],
+            [6..7, '酷使 各PCは最大HP-4。10,000cdtを得る。'],
+            [8..9, '売却 各PCはオプション以外の兵装を一つずつ廃棄。10,000cdtを得る。'],
+          ]
+        ),
+        # DiceTable::Tableが現状1D9に未対応
+        "SPE" => DiceTable::Table.new(
+          "特殊戦況表",
+          "1D10",
+          [
+            '0:雨 雨は砂を土へと変えてしまう。R中、全ユニット移動/突撃不可。',
+            '1:混戦 以下のエリアのユニットをシャッフルする。♠：A⇔C ♣：B⇔D ♦：A⇔D ♥：B⇔C',
+            '2:乱戦 R中、すべての攻撃は［距離：○］になる。',
+            '3:逸失 敵拠点エリアのユニットを［♠♣：A ♦♥：B］に移動。味方拠点エリアのユニットを［♠♣：D ♦♥：C］に移動。',
+            '4:突風 艦船、オブジェクト以外の全ユニットを「風向き」方向に移動。',
+            '5:流砂 以下のエリアのユニットは脱出のため、MPとEPを［3］点失う。［♠：A ♣：B ♦：C ♥：D］',
+            '6:混乱 母船内でトラブル発生。R中、【整備】は行えない。［♠♥：味方側 ♣♦：敵側］',
+            '7:岩盤 以下のエリアのユニットは岩盤に乗り上げ、《クリティカル》が1回発生。［♠：A ♣：B ♦：C ♥：D］',
+            '8:混乱 R中、イニシア値を逆順で処理する。 ※【エイミング】等の高低も逆として処理する。',
+            '9:飛礫 飛礫によって、すべてのユニットはAPを［1D10］（以下のエリアでは［2D10］）点失う。［♠：A ♣：B ♦：C ♥：D］',
+          ]
+        ),
+      }.freeze
+      register_prefix('-?\d*AG', 'OM', 'NM', TABLES.keys, SECOND_ED_TABLES.keys)
```

Comment:
```suggestion
      register_prefix_from_super_class()
      register_prefix(SECOND_ED_TABLES.keys)
```

親クラスのprefixを引き継げる [`BCDice::Base.register_prefix_from_super_class`](https://github.com/bcdice/BCDice/blob/825b39928a708afd1fd5e33a62a5a1bf70ba0450/lib/bcdice/base.rb#L27-L29) を使うのが良さそうです。

Reply:
ありがとうございます。こちらも修正します。